CHANGELOG.md

```diff
@@ -16,6 +16,44 @@ accidentally triggering the load of a previous DB version.**
 **Thanks**
 * @zeeshanshabbir93 for reporting an issue with joins
 
+## 1.7.5 (2021-02-12)
+
+This maintenance release contains bugfixes since the 1.7.4 release.
+Most of these fixes were backported from the 2.0.0 and 2.0.1 releases.
+We deem it high priority for upgrading for users on TimescaleDB 1.7.4
+or previous versions.
+
+In particular the fixes contained in this maintenance release address
+issues in continuous aggregates, compression, JOINs with hypertables,
```

Comment:
What are the cagg-related changes? "Fix support for complex aggregate expression"?

Reply:
Yes.